containers/base/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.8-alpine3.12
+
```

Comment:
```suggestion

LABEL maintainer="Chiara Rasi <chiara.rasi@scilifelab.se>"

```
You could also add some more labels here to describe the container. Have a look at @jemten and @henrikstranneheim examples in MIP for inspiration

Reply:
I'll check, thanks!